services/subscriber.py

```diff
@@ -20,11 +20,11 @@ def get_subscribers(self, targetings, hours_whitelist, volume):
         if timezones:
             targetings.append({
                 "field": "timezone",
-                "operator": "AND",
+                "operator": "IN",
                 "value": timezones
             })
         try:
-            s = Search(using=es, index="subscribers")
+            s = Search(using=es, index="users")
             s.extra(size=volume, boost_mode='replace')
```

Comment:
this code (with `.extra(size=)`)
```
client = Elasticsearch(["http://localhost:9200"])

targetings = [
    {
        "field": "a",
        "operator": "IN",
        "value": [10, 5, 1, 2, 3]
    }
]

s = Search(using=client, index="subs")

s.extra(size=666)

logical_operator_mappings = {
    'IN': 'must',
    'NOT IN': 'must_not',
}

q = Q()

for query in targetings:
    bool_q = Q('bool',
                **{logical_operator_mappings.get(
                    query.get('operator')
                ): Q('terms', **{query["field"]: query["value"]})})
    q += bool_q

s = s.query(q)

s.query = dslq.FunctionScore(
    query=s.query,
    functions=[dslq.SF('random_score')],
    boost_mode="replace"
    )

# s = s[1]

print(s.to_dict())
```
produces this output 
```
{'query': {'function_score': {'query': {'bool': {'must': [{'terms': {'a': ['10', '5', 1, 2, 3]}}]}}, 'functions': [{'random_score': {}}], 'boost_mode': 'replace'}}}
```
nothing about pagination


and if I use index operator `[]`
```
s = s[1]
print(s.to_dict())
```
then it adds `size`
```
{'query': {'function_score': {'query': {'bool': {'must': [{'terms': {'a': [10, 5, 1, 2, 3]}}]}}, 'functions': [{'random_score': {}}], 'boost_mode': 'replace'}}, 'from': 1, 'size': 1}
```
why is it so..

you can examine yourself the query with `print(s.to_dict())`

may be it's version issue ?
```
elasticsearch (5.4.0)
elasticsearch-dsl (5.3.0)
```